js/ongaku.js

```diff
@@ -263,17 +280,41 @@ window.addEventListener("load", function () {
 	}
 
 	// function to play tracks
-	var order = shuffle(osts.concat(openings).concat(endings));
-	function play () {
+	function play (order) {
 		music.src = order[0].link;
 		music.play();
 		updatePlayButton();
 		displayTrackName();
 		document.body.style.backgroundImage = "url('" + order[0].img + "')";
 	}
 
-	play();
+	function init_track_list(order) {
+		var track_list = '';
+		for(var i = 0; i < order.length; i++) {
+			track_list += '<li>'+order[i].name+'</li>';
+		}
+
+		document.querySelector('#track-list ul').innerHTML = track_list;
+		$('#track-list ul li').on('click', select_track);
+	}
+
+	function select_track(e) {
+		//finding the selected title
+		var selected_title = e.target.innerHTML;
```

Comment:
can you please use let instead ?

Reply:
Sure thing! I'm late to the game on all the es6 hotness haha 🔥